docker-image/Dockerfile.amd64

```diff
@@ -3,17 +3,12 @@ MAINTAINER Shaun Crampton <shaun@tigera.io>
 
 # Since our binary isn't designed to run as PID 1, run it via the tini init daemon.
 ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static-amd64 /sbin/tini
```

Comment:
why not do this in `scratch`? (why start from debian-slim?)

Reply:
Closing this comment in favour of the [discussion above](https://github.com/projectcalico/typha/pull/256#discussion_r267105614).